README.md

```diff
@@ -2,9 +2,54 @@
 
 > Shoppers' favorite sale is always the day after Thanksgiving sale, a.k.a. Black Friday.
 
-![](bots.png)
+![bots shopping to the death](bots.png)
 
-## Pelin kulku
+## Running the server
+
+1. `cd server/`
+2. Prepare the frontend: `npm install && npm run-script build`
+3. Run the backend: `mvn spring-boot:run`
+
+After this, the GUI can be accessed from http://localhost:8080/
+
+## Development
+
+You can develop the frontend with npm run-script watch, but the backend
+does not support hot-reloading.
```

Comment:
Running in an IDE (especially with debugging on) is a good way to do development (all modern IDEs allow reloading of static resources and usually also hot-swapping of Java class changes).

http://docs.spring.io/spring-boot/docs/current/reference/html/howto-hotswapping.html